frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -1677,6 +1683,75 @@ class Worksheet extends React.Component {
             },
         );
     };
+
+    showBundleContent = () => {
+        this.setState({ openedDialog: DIALOG_TYPES.OPEN_CREATE_CONTENT });
+    };
+    showBundleContentCallback = (path) => () => {
+        // Default path for content block is '/'
+        path = path || '/';
+        let validBundles = [];
+        let showContentCounts = 0;
+        let tableIDs = Object.keys(this.showContentCallbacks).sort();
+        tableIDs.forEach((tableID) => {
+            let showContentBundleCallback = this.showContentCallbacks[tableID];
+            let bundlesChecked = showContentBundleCallback();
+            bundlesChecked.forEach((bundle) => {
+                if (bundle.name === '<invalid>') {
+                    return;
+                }
+                validBundles.push(bundle.rawIndex);
+                showContentCounts += 1;
+            });
+        });
+
+        let newSource = [...this.state.ws.info.source];
+
+        // Add %display line to the worksheet source right before each checked bundle
+        validBundles.forEach((index, i) => {
+            const items = ['% display contents ' + path, newSource[index], ''];
+            newSource = [
+                ...newSource.slice(0, index + 3 * i + 1),
+                ...items,
+                ...newSource.slice(index + 3 * i + 1),
+            ];
+        });
+        const toastString =
+            showContentCounts > 0
+                ? 'Show contents for ' +
```

Comment:
Let's use a [template literal](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) for clarity / conciseness. So that would look like:

```
`Show contents for ${showContentCounts} bundle...`
```

Reply:
good suggestion!